cardinal/ecs/world_logger.go

```diff
@@ -0,0 +1,110 @@
+package ecs
+
+import (
+	"github.com/rs/zerolog"
+	"path/filepath"
+	"pkg.world.dev/world-engine/cardinal/ecs/component"
+	"pkg.world.dev/world-engine/cardinal/ecs/storage"
+	"reflect"
+	"runtime"
+)
+
+type WorldLogger struct {
+	world  *World
+	logger *zerolog.Logger
+}
+
+func NewWorldLogger(logger *zerolog.Logger, world *World) WorldLogger {
+	return WorldLogger{
+		logger: logger,
+		world:  world,
+	}
+}
+
+func (wl *WorldLogger) Log(traceId string, logLevel zerolog.Level, message string) {
```

Comment:
It looks like the message parameter is unused here. That seems important.

Reply:
Yeah that's a bug will add unit tests for these.